app/src/main/java/ui/settings/about/PersonAdapter.kt

```diff
@@ -107,13 +105,16 @@ class PersonAdapter : BaseRecyclerViewAdapter(), KoinComponent {
 
                 // LinkedIn
                 linkedIn.setOnClickListener {
-                    ga.sendEvent("About", "LinkedIn", person.name)
+                    FirebaseAnalytics.getInstance(it.context)
```

Comment:
If you want you can pass the context to PersonAdapter and get the instance once directly